### PR TITLE
[MVP-25] Add landing page WIP with landing page layout

### PR DESCRIPTION
**Pull request by alexFaunt**
Adds a /landing-wip page with some placeholder content

This uses a new "landing" layout file, which when we switch over would be renamed default. This adds header / menu / footer etc. 

It also currently also controls font-size, but i'd move that to global styles once we switch over.

The layout is all based on rems / screen width. It's using a philosophy that we have 2 layouts (narrow or mobile/wide or desktop), which is easier to design for, and the font-size is variable within bounds to try and make those two layouts look good at all screen sizes 

This is how that looks with the placeholder content - will make more sense with some actual content:

Sep-16-2021 13-40-28
 
Next jobs: 
1, Add typescript checking - gatsby supports typescript but we're not actually checking it so i know i've violated a bunch of rules
2, Add some content + sort out common typography sizes for content
3, Fix header on scroll
4, Rest of content
5, Update footer
6, Animate menu / add hover states

### CODE REVIEW

File: src/pages/landing-wip.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (igoratron) [inline src/pages/landing-wip.tsx:1]: 🧠 wondering if it's useful to take the approach from [here](https://github.com/gatsbyjs/gatsby/issues/17831#issuecomment-534482935) as opposed to creating a `/landing-wip` page in production

File: src/components/layouts/landing.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: ${hillsSize};
+`;
+
+const Content = styled.div``;
+
+const footerLinks = {
+  privacy: 'todo',
```
Comment (igoratron) [inline src/components/layouts/landing.tsx:30]: 📝 this could be `/legal/privacy`

**Review by igoratron — APPROVED** [review]

File: src/pages/landing-wip.tsx (hunk 1 above)
Comment (alexFaunt) [reply src/pages/landing-wip.tsx:1]: Cool yeah i'll try the onCreatePage one and have a wip folder that gets removed on prod

Comment (alexFaunt) [reply src/pages/landing-wip.tsx]: done, url is now localhost:8000/wip/landing but on production no pages under /wip/ get built

**Pull request merged**